curator-framework/src/test/java/org/apache/curator/framework/imps/TestFrameworkEdges.java

```diff
@@ -670,9 +670,13 @@ public void testFailure() throws Exception
             client.checkExists().forPath("/hey");
             fail();
         }
+        catch ( KeeperException.SessionExpiredException e )
+        {
+            // correct, this happens on ZK 3.6.3+
```

Comment:
Why? Shall we setup a link to the reason?

It is a bit weird a patch version bump cause behavior changes.

Reply:
I really do not know why this is happening.
I have looked thru ZK 3.6.3 release notes and I haven't found anything that may be releated.

Reply:
Fine. It is not a blocker for client library such as Curator, but is interesting...

Reply:
probably Session Expiration is happening earlier with 3.6.3
with 3.6.2 we still see a ConnectionLoss, that usually becomes a SessionExpired when the client is able to connect again and the ZK server already expired it on the server side